charts/opentelemetry-collector/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## OpenTelemetry Collector
 
+### v0.98.6 / 2024-12-02
+- [Fix] Target Alloctor services always being created with.
```

Comment:
Maybe we can improve the changelog entry? Sounds a bit confusing to me.

Reply:
Sure, do you have any suggestion?

Reply:
I would suggest something like:

```suggestion
- [Fix] Create the Target Allocator service only when applicable. 
```